monitoring/dashboard.py

```diff
@@ -525,6 +542,111 @@ def top10_errors_by_bucket(title, code):
     title="5xx : Top10 by Bucket", code='~"5.."'
 )
 
+
+quotaHealth = GaugePanel(
+    title="Quota service uptime",
+    dataSource="${DS_PROMETHEUS}",
+    calc="mean",
+    format=UNITS.PERCENT_FORMAT,
+    min=0,
+    max=100,
+    noValue="-",
+    targets=[Target(
+        expr="\n".join([
+            'sum(rate(s3_utilization_service_state{namespace="${namespace}", state="healthy"}[$__rate_interval])) * 100',  # noqa: E501
+            "   /",
+            'sum(rate(s3_utilization_service_state{namespace="${namespace}", state=~"^(unhealthy|healthy)$"}[$__rate_interval]))',  # noqa: E501
```

Comment:
For promql I think you don't need `^$` in regex:

> Regex matches are fully anchored. A match of env=\~"foo" is treated as env=~"^foo$".
https://prometheus.io/docs/prometheus/latest/querying/basics/